src/test/java/net/sf/jabref/logic/cleanup/CleanupWorkerTest.java

```diff
@@ -56,12 +56,11 @@ public void setUp() throws IOException {
             Globals.journalAbbreviationLoader = mock(JournalAbbreviationLoader.class);
         }
 
-        if (Globals.protectedTermsLoader == null) {
-            Globals.protectedTermsLoader = new ProtectedTermsLoader(
-                    new ProtectedTermsPreferences(ProtectedTermsLoader.getInternalLists(), Collections.emptyList(),
-                            Collections.emptyList(), Collections.emptyList()));
-            ProtectTermsFormatter.setProtectedTermsLoader(Globals.protectedTermsLoader);
-        }
+        ProtectedTermsLoader protectedTermsLoader = new ProtectedTermsLoader(
+                new ProtectedTermsPreferences(ProtectedTermsLoader.getInternalLists(), Collections.emptyList(),
+                    Collections.emptyList(), Collections.emptyList()));
+        Assert.assertNotEquals(Collections.emptyList(), protectedTermsLoader.getProtectedTerms());
+        ProtectTermsFormatter.setProtectedTermsLoader(protectedTermsLoader);
```

Comment:
instead of the setProtectedTermsLoader use `termsFormatter = new ProtectTermsFormatter(protectedTermsLoader)` and use it below in cleanupCasesAddsBracketAroundAluminiumGalliumArsenid.